### DISCUSSION TITLE
Backward splat

### DISCUSSION THREAD

**Discussion by spacepolice10** [remix-run/remix#9170] [Q&A] [answered]
Hi there,

I know about the patter like `some-route/*`. In this case everything after *some-route* will be considered a route. What if I want to achieve similar pattern but for another side of end of the path. Like `*/create-item` which will catch any existent routes and render `create-item` inside their `<Outlet />`. Is something like this possible?

**Comment by sergiodxa** [comment, marked as answer]
No with the current router, but you could write an RFC to support it, it could be an interesting feature.
